.rubocop.yml

```diff
@@ -3,15 +3,8 @@ AllCops:
   DisplayCopNames: true
   ExtraDetails: true
   Exclude:
-    - 'lib/truffle/date.rb'
-    - 'lib/truffle/pathname.rb'
-    - 'lib/truffle/securerandom.rb'
-    - 'lib/truffle/socket.rb'
-    - 'lib/truffle/stringio.rb'
-    - 'lib/truffle/strscan.rb'
-    - 'lib/truffle/timeout.rb'
-    - 'lib/truffle/date/**/*.rb'
-    - 'lib/truffle/socket/**/*.rb'
+    - 'lib/truffle/date/format.rb' # crashes Rubocop in VariableForce
```

Comment:
This causes a crash. I'm not sure if it's worth trying to update or open an issue.